src/main/java/datameer/awstasks/ant/ec2/AbstractEc2Task.java

```diff
@@ -48,8 +50,16 @@ public void setRegion(String region) {
 
     private AmazonEC2 createEc2() {
         AmazonEC2Client ec2Client = new AmazonEC2Client(new BasicAWSCredentials(_accessKey, _accessSecret));
+        Level old = Logger.getRootLogger().getLevel();
+        Region region;
+        try {
+            Logger.getRootLogger().setLevel(Level.ERROR);
+            region = Region.getRegion(Regions.valueOf(_region.toUpperCase()));
+        } finally {
+            Logger.getRootLogger().setLevel(old);
```

Comment:
This will override the setting of the level to error 4 lines before.

Reply:
@mschaaf : changes made